controller/Controller.class.php

```diff
@@ -148,13 +148,22 @@ protected static function getRouterWithRoutes(): \Routing\RouteCollector
                 Response::enableHttpCache();
                 return ['page/' . $slug, []];
             } else {
-                return NavActions::execute404();
+                return static::checkLowerUri($slug);
             }
         });
 
         return $router;
     }
 
+    protected static function checkLowerUri($uri)
```

Comment:
This function has unclear side effects. One would not reasonably expect `checkLowerUri` to re-route to a 404.

There is likely a better name than this, but if you can't think of anything better, name it `doLowerUriRedirectOr404`.